package/kedro_viz/models/flowchart.py

```diff
@@ -424,6 +424,8 @@ class ModularPipelineNode(GraphNode):
     # The type for Modular Pipeline Node
     type: str = GraphNodeType.MODULAR_PIPELINE.value
 
+    tags: Set[str] = Field(default_factory=set)
```

Comment:
tags is already part of the parent (i.e., GraphNode). I don't think we need this here.

Reply:
Done.